internal/receiver/signalfxgatewayprometheusremotewritereceiver/server.go

```diff
@@ -38,13 +38,13 @@ type prometheusRemoteWriteServer struct {
 }
 
 type serverConfig struct {
-	confighttp.ServerConfig
 	component.TelemetrySettings
 	Reporter reporter
 	component.Host
 	Mc     chan<- pmetric.Metrics
 	Parser *prometheusRemoteOtelParser
 	Path   string
+	confighttp.ServerConfig
```

Comment:
Not a problem at all, but curious as to why this line was moved to the end of the struct?

Reply:
> Not a problem at all, but curious as to why this line was moved to the end of the struct?

Not sure. It was just the output from `make generate-metrics`: https://github.com/signalfx/splunk-otel-collector/pull/5001/commits/bc8fa8ce02010c2941f18c74580c2be461b895bb